### ISSUE TITLE
Create an example cross section model that uses a lookup file.

### ISSUE THREAD

**Issue by abuts** [mantidproject/mantid#7535]
This issue was originally [TRAC 6689](http://trac.mantidproject.org/mantid/ticket/6689)

This ticket is blocked by : 
- [X] [TRAC8060](http://trac.mantidproject.org/mantid/ticket/8060)

This is VATES workfolw request related to Tobyfit integration with Mantid

**Comment by AnthonyLim23** [comment]
This is no longer needed.
